p2p/protocol/identify/id_test.go

```diff
@@ -356,7 +356,7 @@ func TestIdentifyDeltaOnProtocolChange(t *testing.T) {
 	defer cancel()
 
 	h1 := blhost.NewBlankHost(swarmt.GenSwarm(t))
-	h2 := blhost.NewBlankHost(swarmt.GenSwarm(t))
+	h2 := blhost.NewBlankHost(swarmt.GenSwarm(t, swarmt.OptDisableQUIC))
```

Comment:
What's the reason we're disabling QUIC? Are there any problems with parallel connections?

Reply:
No, I disabled it to inspect the bytes on the wire. This is a remnant of that